_DeploymentAndDistroPackaging/ansible/group_vars/all

```diff
@@ -1,4 +1,5 @@
 ---
+workdir: /tmp/ciao
```

Comment:
<b>What are the permissions on the ciao directory if it is created?</b>
Default permissions. I can add `mode=600` to make it only readable by the user running ansible.

<b> What happens if it is already present and globally readable before the start of the ansible script?</b>
If I use `mode=600` and the directory already exists, the permissions will be changed to 600

<b>Do you leak the private certificates to other users on the deployment machine?</b>
I didn't realize but yes.